tests/test_install.py

```diff
@@ -135,7 +135,8 @@ def cli(self, *args, **kwargs):
 
     def setup_py(self, *args, **kwargs):
         with chdir(self.pkg_path):
-            args = ["python", "setup.py"] + list(args)
+            args = ["python", "-Wignore", "setup.py"] + list(args)
```

Comment:
I suspect something like the following warning is happening in windows and causing the [error in the CI](https://cirrus-ci.com/task/5380876931432448?command=test#L672):
```
c:\python38\lib\site-packages\setuptools\distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  warnings.warn(
```
But I could not replicate it myself...
So I hope ignoring the error might fix the assertion 🤞